app/javascript/lib/UseCases/Board/Provinces/Factory/Build.js

```diff
@@ -0,0 +1,12 @@
+export default class Build {
+  constructor(province) {
+    this.province = province;
+  }
+
+  buildArmsFactory() {
+    this.province.hasArmsFactory = true;
+  }
+  buildNavalFactory() {
+    this.province.hasNavalFactory = true;
```

Comment:
At some point we're going to run up against the discrepancy that I was calling these `shipyards` before. I kinda prefer the symmetry of `arms factory` and `naval factory` so it may be worth switching over. But just a heads up!

Reply:
Yeah, I think for consistency we won't be switching the ids over, and will be keeping the term `armaments/shipyards`, but at least logic-wise from now on whenever we refer to a province, we can call them `Arms/NavalFactory`